pisa/core/detectors.py

```diff
@@ -30,7 +30,7 @@
 from pisa.utils.hash import hash_obj
 from pisa.utils.log import set_verbosity, logging, Levels
 from pisa.utils.random_numbers import get_random_state
-from pisa.analysis.analysis import update_param_values_detector
+#from pisa.analysis.analysis import update_param_values_detector
```

Comment:
If that causes problems we don't have to import the function from analysis, but then we have to define it here again. Otherwise the unit test will fail (what it currently does).

Reply:
there was no issue in the past with this import loop? then I'm ok to just put it back. i took the function out from the unit test too.

Reply:
Hmm, I can not remember having issues with that, but I can see that it is not ideal. The problem is that when we move the function we also have to move the original update_params function out of analysis (maybe to distribution_maker.py), because we need it here.